spec/frontman/context_spec.rb

```diff
@@ -25,19 +25,29 @@
     end
 
     it 'should return not react to unknown content' do
-      expect(subject.content_for?('unknown content')).to eq false
+      key = 'unknown content'
+
+      expect(subject.content_for?(key)).to eq false
+
+      expect { subject.append_content(key) }.to raise_error NoMethodError
```

Comment:
I think we should change the behavior of the `append_content` method when there's no content block yet for the key the user provides.

The situation where you don't want to append content _unless_ the block exists will, I believe, occur less often than the situation where you want to append the content regardless of whether the block exists. For the first scenario, you could still use an inline check (e.g. `append_content(:foo, 'bar') unless content_for?(:foo)` to prevent his from happening.

Therefore, I think we should make the `append_content` method so that it automatically creates the content block if there's no block yet, instead of it throwing an error. What do you think?

Reply:
Yes I think its better to have it create the content block if not existing. I have updated the code accordingly.